src/index.html

```diff
@@ -17,16 +17,29 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
-
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="form-search-bar form-search-bar--big"
+      data-qa="big"
+    >
+      <input
+        class="input-bar input-bar--big"
+        id="input-icon-big"
+        type="text"
+        data-qa="keypress"
+        placeholder="Try “Los Angeles”"
+      />
+    </form>
+    <form
+      class="form-search-bar form-search-bar--small"
+      data-qa="small"
+    >
+      <input
+        class="input-bar input-bar--small"
+        id="input-icon-small"
+        type="text"
+        data-qa="hover small"
```

Comment:
Similar to the previous form, the `data-qa` attribute is used here. Verify its purpose and ensure it aligns with your project's requirements or testing framework.